lib/mocha.js

```diff
@@ -290,6 +292,39 @@ Mocha.prototype.loadFiles = function(fn) {
   fn && fn();
 };
 
+/**
+ * Removes a previously loaded file from Node's `require` cache.
+ *
+ * @private
+ * @see {@link Mocha#unloadFiles}
+ * @param {string} file - Pathname of file to be unloaded.
+ */
+Mocha.prototype.unloadFile = function(file) {
```

Comment:
Yeah. Decided to mark existing scheme @private for moment, but since `Mocha#unloadFiles` won't necessarily fix everything, this leaves a backdoor JIC.